### PR TITLE
Wilderness NPC and item spawns.

### PR DESCRIPTION
**Pull request by HarleyGilpin**
Manually added all NPC and item spawns for the wilderness and some wilderness underground dungeon areas.

### CODE REVIEW

**Comment by alycii** [comment]
In the future, please use semantic commit messages. You can find more information on them here:

https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

File: game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/spawns/spawns_10057.plugin.kts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * WILDERNESS NPC & ITEM SPAWNS
+ * @author Harley Gilpin <https://github.com/HarleyGilpin>
+ */
+spawn_npc(npc = Npcs.KOLODION, x = 2541, z = 4716, walkRadius = 3, direction = Direction.WEST)
+spawn_npc(npc = Npcs.GUNDAI, x = 2534, z = 4713, walkRadius = 3, direction = Direction.WEST)
+spawn_npc(npc = Npcs.LUNDAIL, x = 2535, z = 4719, walkRadius = 0, direction = Direction.SOUTH)
```
Comment (alycii) [inline game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/spawns/spawns_10057.plugin.kts:7]: NPCs that don't walk do not need their walkRadius set to 0, simply not having it there will make them stationary. To finish the review, please fix any issues by removing `walkRadius = 0` :)

**Review by alycii — CHANGES_REQUESTED** [review]

**Review by HarleyGilpin — COMMENTED** [review]
Thanks for the feedback, I've fixed the identified issue with NPC walkRadius parameter value being set superfluously.

**Review by alycii — APPROVED** [review]
Quite a lot of data! Nice work 👍

**Pull request merged**